<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncounterWidget</class>
 <widget class="QWidget" name="EncounterWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>178</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QComboBox" name="combo_type">
     <item>
      <property name="text">
       <string>Land</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Cave</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Water</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>BugContest</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>OldRod</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GoodRod</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SuperRod</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RockSmash</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HeadbuttLow</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HeadbuttHigh</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LandDay</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LandNight</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LandMorning</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LandAfternoon</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LandEvening</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CaveDay</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CaveNight</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CaveMorning</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CaveAfternoon</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>CaveEvening</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WaterDay</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WaterNight</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WaterMorning</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WaterAfternoon</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>WaterEvening</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3" rowspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spin_step_chance"/>
   </item>
   <item row="3" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Slots</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="encounter_layout"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Step Chance:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="button_add_slot">
     <property name="text">
      <string>Add Slot</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="button_remove_encounter">
     <property name="text">
      <string>Remove this encounter</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
